tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
-envlist = py35,py36,py37,py38,pypy3
+envlist = py36,py37,py38,pypy3
 
 [testenv]
 deps=
     pytest
```

Comment:
pytest-cov already requires pytest, then you can remove this line

Reply:
I think it's better to write all dependencies we use for testing, not relying on dep tree.